src/logic/map_objects/tribes/worker.h

```diff
@@ -77,6 +77,10 @@ class Worker : public Bob {
 	explicit Worker(const WorkerDescr&);
 	~Worker() override;
 
+	Player& owner() const {
```

Comment:
Why have you added this method?
Are you adding it intentionally (okay by me) or is it "added" because it has been removed from master since starting this branch?

Reply:
Removed.